docker-compose/pattern-1/mysql/scripts/mysql5.7.sql

```diff
@@ -1,1154 +1,1153 @@
-            REFRESH_TOKEN VARCHAR(255),
-            CONSUMER_KEY_ID INTEGER,
-            AUTHZ_USER VARCHAR (100),
-            TENANT_ID INTEGER,
-            USER_DOMAIN VARCHAR(50),
-            USER_TYPE VARCHAR (25),
-            GRANT_TYPE VARCHAR (50),
-            TIME_CREATED TIMESTAMP DEFAULT 0,
-            REFRESH_TOKEN_TIME_CREATED TIMESTAMP DEFAULT 0,
-            VALIDITY_PERIOD BIGINT,
-            REFRESH_TOKEN_VALIDITY_PERIOD BIGINT,
-            TOKEN_SCOPE_HASH VARCHAR(32),
-            TOKEN_STATE VARCHAR(25) DEFAULT 'ACTIVE',
-            TOKEN_STATE_ID VARCHAR (128) DEFAULT 'NONE',
-            SUBJECT_IDENTIFIER VARCHAR(255),
-            PRIMARY KEY (TOKEN_ID),
-            FOREIGN KEY (CONSUMER_KEY_ID) REFERENCES IDN_OAUTH_CONSUMER_APPS(ID) ON DELETE CASCADE,
-            CONSTRAINT CON_APP_KEY UNIQUE (CONSUMER_KEY_ID,AUTHZ_USER,TENANT_ID,USER_DOMAIN,USER_TYPE,TOKEN_SCOPE_HASH,
-                                           TOKEN_STATE,TOKEN_STATE_ID)
+CREATE TABLE IF NOT EXISTS REG_CONTENT_HISTORY (
+             REG_CONTENT_ID INTEGER NOT NULL,
+             REG_CONTENT_DATA LONGBLOB,
+             REG_DELETED   SMALLINT,
+             REG_TENANT_ID INTEGER DEFAULT 0,
+             CONSTRAINT PK_REG_CONTENT_HISTORY PRIMARY KEY(REG_CONTENT_ID, REG_TENANT_ID)
 )ENGINE INNODB;
 
-CREATE INDEX IDX_AT_CK_AU ON IDN_OAUTH2_ACCESS_TOKEN(CONSUMER_KEY_ID, AUTHZ_USER, TOKEN_STATE, USER_TYPE);
+CREATE TABLE IF NOT EXISTS REG_RESOURCE (
+            REG_PATH_ID         INTEGER NOT NULL,
+            REG_NAME            VARCHAR(256),
+            REG_VERSION         INTEGER NOT NULL AUTO_INCREMENT,
+            REG_MEDIA_TYPE      VARCHAR(500),
+            REG_CREATOR         VARCHAR(31) NOT NULL,
+            REG_CREATED_TIME    TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP,
+            REG_LAST_UPDATOR    VARCHAR(31),
+            REG_LAST_UPDATED_TIME    TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP,
+            REG_DESCRIPTION     VARCHAR(1000),
+            REG_CONTENT_ID      INTEGER,
+            REG_TENANT_ID INTEGER DEFAULT 0,
+            REG_UUID VARCHAR(100) NOT NULL,
+            CONSTRAINT PK_REG_RESOURCE PRIMARY KEY(REG_VERSION, REG_TENANT_ID)
+)ENGINE INNODB;
 
-CREATE INDEX IDX_TC ON IDN_OAUTH2_ACCESS_TOKEN(TIME_CREATED);
+ALTER TABLE REG_RESOURCE ADD CONSTRAINT REG_RESOURCE_FK_BY_PATH_ID FOREIGN KEY (REG_PATH_ID, REG_TENANT_ID) REFERENCES REG_PATH (REG_PATH_ID, REG_TENANT_ID);
+ALTER TABLE REG_RESOURCE ADD CONSTRAINT REG_RESOURCE_FK_BY_CONTENT_ID FOREIGN KEY (REG_CONTENT_ID, REG_TENANT_ID) REFERENCES REG_CONTENT (REG_CONTENT_ID, REG_TENANT_ID);
+CREATE INDEX REG_RESOURCE_IND_BY_NAME USING HASH ON REG_RESOURCE(REG_NAME, REG_TENANT_ID);
+CREATE INDEX REG_RESOURCE_IND_BY_PATH_ID_NAME USING HASH ON REG_RESOURCE(REG_PATH_ID, REG_NAME, REG_TENANT_ID);
+CREATE INDEX REG_RESOURCE_IND_BY_UUID USING HASH ON REG_RESOURCE(REG_UUID);
+CREATE INDEX REG_RESOURCE_IND_BY_TENAN USING HASH ON REG_RESOURCE(REG_TENANT_ID, REG_UUID);
```

Comment:
Shouldn't this index name be 'REG_RESOURCE_IND_BY_TENANT' rather than 'REG_RESOURCE_IND_BY_TENAN'?

Reply:
A good point Chiranga! Here we have used the database scripts shipped with the product. If this can be fixed may be we need to that in the product release. Otherwise, when this script get updated, we may need to fix it again and again in this repository. Thanks!